src/components/editor/spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import test from 'tape';
+import EditorFactory from './';
+import UpstreamEditor from 'draft-js-plugins-editor';
+import getShallowInstance from 'utils/shallow';
+
+const getState = (el) => {
+  if (el.type === 'span') {
+    return el.props.children;
+  }
+};
+
+test('Editor', t => {
+  const Editor = EditorFactory(React);
+  t.plan(2);
+
+  let instance = getShallowInstance(<Editor />);
+
+  t.equals(getState(instance.props.children[0]), 'Reading');
+  t.equals(instance.props.children[1].type, UpstreamEditor);
```

Comment:
We should be shallow rendering with enzyme; I thought it was set up on master, but it is not. Just `npm i -D enzyme` and you're good to go. Sorry for the confusion.

Also, more broadly, I want to avoid rendering into the DOM to test anything. So let's try to find the shallow equivalent before we render all the way out. Feel free to highlight any blocks you might have trouble with and I'll provide suggestions/approaches.